project_tests.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+if [ ! -e "venv/bin/python3.5" ]; then
+    echo "could not find venv/bin/python3.5, recreating venv"
+    rm -rf venv
+    virtualenv --python=python3.5 venv
+fi
+
```

Comment:
could probably use `tox` to run on multiple versions?

Reply:
Yes, I have it working on py27 and py35 along with a `tox.ini` will switch up the `project_tests.sh` to just run via `tox`.

Reply:
Have added this now, left placeholders in there for when we want to capture coverage via CI.